_rules/css-restrict-orientation-b33eff.md

```diff
@@ -0,0 +1,265 @@
+
+```html
+<html lang="en">
+	<head>
+		<title>Page with some content</title>
+		<style>
+			body {
+				transform: rotate(90deg);
+			}
+			@media (orientation: portrait) {
+				body {
+					transform: rotate(90deg);
+				}
+			}
+		</style>
+	</head>
+	<body>
+		<main>
+			Page Content
+		</main>
+	</body>
+</html>
+```
+
+### Failed
+
+#### Failed Example 1
+
+A page where CSS [transform](https://www.w3.org/TR/css-transforms/#propdef-transform) property has [rotate](https://www.w3.org/TR/css-transforms/#funcdef-transform-rotate) [transform function](https://www.w3.org/TR/css-transforms/#transform-functions) conditionally applied on the [orientation](https://www.w3.org/TR/css3-mediaqueries/#orientation) [media feature](https://www.w3.org/TR/css3-mediaqueries/#media1) which restricts the element to `landscape` orientation.
+
+```html
+<html lang="en">
+	<head>
+		<title>Page with some content</title>
+		<link rel="stylesheet" href="../test-assets/b33eff/style.css" />
+	</head>
+	<body>
+		Page Content
+	</body>
+</html>
+```
+
+#### Failed Example 2
+
+A page where CSS [transform](https://www.w3.org/TR/css-transforms/#propdef-transform) property has [matrix3d](https://drafts.csswg.org/css-transforms-2/#funcdef-matrix3d) [transform function](https://www.w3.org/TR/css-transforms/#transform-functions) conditionally applied on the [orientation](https://www.w3.org/TR/css3-mediaqueries/#orientation) [media feature](https://www.w3.org/TR/css3-mediaqueries/#media1) which restricts the element to `portrait` orientation.
+
+```html
+<html lang="en">
+	<head>
+		<title>Page with some content</title>
+		<style>
+			@media (orientation: landscape) {
+				body {
+					transform: matrix3d(0, -1, 0, 0, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1);
+				}
+			}
+		</style>
+	</head>
+	<body>
+		Page Content
+	</body>
+</html>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+A page where there are CSS styles.
+
+```html
+<html lang="en">
+	<head>
+		<title>Page with some content</title>
+	</head>
+	<body>
+		I am a page with no styles
+	</body>
+</html>
+```
+
+#### Inapplicable Example 2
+
+A page that has no CSS [transform](https://www.w3.org/TR/css-transforms/#propdef-transform) property specified.
+
+```html
+<html lang="en">
+	<head>
+		<title>Page with some content</title>
+		<style>
+			html {
+				font-size: 22px;
+			}
+			@media (min-width: 30em) {
+				font-size: 100%;
+			}
+		</style>
+	</head>
+	<body>
+		Page Content
+	</body>
+</html>
+```
+
+#### Inapplicable Example 3
+
+A page where CSS [transform](https://www.w3.org/TR/css-transforms/#propdef-transform) property is applied to an element that is not [visible](#visible).
+
+```html
+<html lang="en">
+	<head>
+		<title>Page with some content</title>
+		<style>
+			@media (orientation: lanscape) {
+				body {
+					transform: rotateZ(0, 0, 1, 270deg);
+				}
+			}
+		</style>
+	</head>
+	<body style="display:none;">
+		<main>
+			Page Content
+		</main>
+	</body>
+</html>
+```
```

Comment:
I am missing an inapplicable example where the transform property is used to rotate the page 90 degrees but not based on the current landscape or portrait orientation.